packages/application-shell/src/components/quick-access/types.ts

```diff
@@ -0,0 +1,34 @@
+import { DocumentNode } from 'graphql';
```

Comment:
This time I decided to extract some of the common types into a separate file, to avoid too much repetition.

The main reason we usually don't do this is because of the babel plugin for generating prop types, as it can only do that when types are within the same file.
However, `<QuickAccess>` is an internal component and is not used "outside" of the library, therefore prop types are not very important.